### DISCUSSION TITLE
Allow centered header card for sections layout

### DISCUSSION THREAD

**Discussion by jhbruhn** [home-assistant/frontend#20049] [Dashboard] [answered]
The new Sections Layout is already really nice, except being just a beta. One seemingly old and by now less used function of dashboards are badges. Other card frameworks, like Mushroom, include an alternative to that, in the form of chips.

To achieve the goal of a more modern and streamlined design, it would be nice to either
a) redesign badges to be more modern and useful, like the mushroom chips are, or
b) allow any card to be an optional header-row card of the section view, which is separate from the sections.

With option b), users could place their own badge-variant, like mushroom chips, onto the header of the layout, and not just the header of for instance the first section.

**Comment by Krzemq** [comment]
The header row would be great. Or an option to set a width for a section, but it could behave differently on mobile/desktop so I think that optional header row with full width would be awesome!

**Comment by jhbruhn** [comment, marked as answer]
a) has been implemented in 2024.8
